src/extras/helpers/BoxHelper.js

```diff
@@ -22,16 +22,10 @@ THREE.BoxHelper.prototype.constructor = THREE.BoxHelper;
 
 THREE.BoxHelper.prototype.update = function ( object ) {
 
-	var geometry = object.geometry;
+	var box = new THREE.Box3().setFromObject(object);
```

Comment:
Yeah. I deleted my earlier comment by mistake, so re-posted it after-the-fact. I wonder if there is an un-delete button?... I could have made good use of it.